### PR TITLE
Changes that make kyber work on STM32

### PR DESCRIPTION
**Pull request by anhu**
(wolfcrypttest and benchmark)

# Testing

Enabled pqm4 and ran benchmarks and wolfcrypttest

### CODE REVIEW

**Comment by anhu** [comment]
Assigned to @dgarske for review as the failure isn't really a failure.

File: wolfcrypt/src/ext_kyber.c [hunk 1]
```diff
@@ -314,6 +313,8 @@ int wc_KyberKey_MakeKey(KyberKey* key, WC_RNG* rng)
     }
 
 #ifdef HAVE_LIBOQS
+    const char* algName = NULL;
+    OQS_KEM *kem = NULL;
```
Comment (dgarske) [inline wolfcrypt/src/ext_kyber.c:317]: You can't move these here. Some compilers will have issue. Please move to top and wrap with `#ifdef HAVE_LIBOQS` or put in brace section.

File: wolfcrypt/src/ext_kyber.c [hunk 2]
```diff
@@ -404,6 +402,8 @@ int wc_KyberKey_Encapsulate(KyberKey* key, unsigned char* ct, unsigned char* ss,
     }
 
 #ifdef HAVE_LIBOQS
+    const char * algName = NULL;
+    OQS_KEM *kem = NULL;
```
Comment (dgarske) [inline wolfcrypt/src/ext_kyber.c:406]: Same

File: wolfcrypt/src/ext_kyber.c [hunk 3]
```diff
@@ -493,6 +491,8 @@ int wc_KyberKey_Decapsulate(KyberKey* key, unsigned char* ss,
...
```
Comment (dgarske) [inline wolfcrypt/src/ext_kyber.c:495]: Same

**Review by dgarske — CHANGES_REQUESTED** [review]

**Review by dgarske — APPROVED** [review]

**Pull request merged**